src/includes/class-wp-cli.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * Network Management Command
+ *
+ */
+class WPMultiNetwork_Command extends WP_CLI_Command {
```

Comment:
Can we prefix this with `WPMN_` to match the other functions and classes please?